code/index.html

```diff
@@ -1,30 +1,181 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Ballet Dance Academy</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Ballet&family=Dancing+Script&family=Open+Sans+Condensed:wght@300&display=swap"
+    rel="stylesheet">
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+</head>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+<body>
+
+  <!-----------  HEADER  ----------->
+  <header>
+    <div class="hero-slide">
+      <div class="slider " id="slider1">
+        <input type="radio" name="slides" checked="checked" id="slides_1" />
```

Comment:
I also tried on my phone but the video does not load there? I had the same problem which I solved with putting: **muted playsinline**  in the video for mobile

Its really nice with the videos but a bit confusing how to navigate between them. With the dots I at least think that you should swipe, but you have to press them. Challenge for next time ;)